File: app/src/main/java/org/michaelbel/moviemade/AboutActivity.java
package org.michaelbel.moviemade;

import android.databinding.DataBindingUtil;
import android.os.Bundle;
import android.support.annotation.Nullable;

import org.michaelbel.moviemade.databinding.ActivityAboutBinding;
import org.michaelbel.moviemade.mvp.base.BaseActivity;
import org.michaelbel.moviemade.ui.fragment.AboutFragment;

public class AboutActivity extends BaseActivity {

    public ActivityAboutBinding binding;

    @Override
    protected void onCreate(@Nullable Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        binding = DataBindingUtil.setContentView(this, R.layout.activity_about);

        setSupportActionBar(binding.toolbar);
        startFragment(new AboutFragment(), binding.fragmentView);
    }
}

File: app/src/main/java/org/michaelbel/moviemade/presentation/features/about/AboutActivity.kt
package org.michaelbel.moviemade.presentation.features.about

import android.os.Bundle
import kotlinx.android.synthetic.main.activity_default.*
import org.michaelbel.moviemade.R
import org.michaelbel.moviemade.core.utils.ViewUtil
import org.michaelbel.moviemade.presentation.base.BaseActivity

class AboutActivity: BaseActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_default)

        setSupportActionBar(toolbar)
        toolbar.setBackgroundColor(ViewUtil.getAttrColor(this, android.R.attr.colorBackground))
        ViewUtil.setElevation(appbar, 0F)

        if (savedInstanceState == null) {
            startFragment(AboutFragment(), container.id)
        }
    }
}